report_xml/tests/test_report_xml.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from lxml import etree
+from odoo.tests import common
+
+
+class TestXmlReport(common.TransactionCase):
+    def test_xml(self):
+        report_object = self.env['ir.actions.report.xml']
+        report_name = 'report_xml.demo_report_xml_view'
+        docs = self.env['res.company'].search([], limit=1)
+        rep = report_object.render_report(
+            docs.ids, report_name, {'report_type': u"qweb-xml"}
```

Comment:
I can see that the usual practice is to call `render_report` not specifying the report_type, as it is looked up inside the method.